BUILD.md

```diff
@@ -5,159 +5,159 @@ Main project target is to release the artifacts to one of central dependencies r
 (Maven central so far, but might be Bintray in the future). Unfortunately there is no so far any consistent way to 
 publish the project using TravisCI, so manual publishing is required.
```

Comment:
@andrii-kovalenko-ct ah ok, it seems that it's an outdated sentence :) because it says: ` but might be Bintray in the future`